src/Umbraco.Web/PublishedPropertyExtension.cs

```diff
@@ -43,22 +43,29 @@ public static T Value<T>(this IPublishedProperty property, string culture = null
 
                 // cannot cast nor convert the value, nothing we can return but 'default'
                 // note: we don't want to fallback in that case - would make little sense
-                return default;
+                return defaultValue;
             }
 
             // we don't have a value, try fallback
             if (PublishedValueFallback.TryGetValue(property, culture, segment, fallback, defaultValue, out var fallbackValue))
+            {
                 return fallbackValue;
+            }
 
             // we don't have a value - neither direct nor fallback
             // give a chance to the converter to return something (eg empty enumerable)
             var noValue = property.GetValue(culture, segment);
             if (noValue is T noValueAsT) return noValueAsT;
+            if (noValue == null)
```

Comment:
Maybe we could put this before the test for noValueAsT?

Reply:
yep sure